components/inputs/input-styles.js

```diff
@@ -114,3 +114,8 @@ export const inputStyles = css`
 		background-image: none;
 	}
 `;
+
+export const inlineHelpStyles = {
```

Comment:
I think we should rely on CSS class instead. We generally don't export styles for inline.
```javascript
export const inlineHelpStyles = css`
    .d2l-input-inline-help {
        marginTop: '0.5rem'
    }
`;
```

Also, I think we should not be specifying `fontSize: '15px'` here. At least for the checkbox case, the font-size will come from `d2l-body-small`, which I think is correct.